.github/workflows/test.yml

```diff
@@ -80,6 +80,22 @@ jobs:
         name: Test
         run: |
           make -o build test-unit
+      -
+        name: Prepare reports
+        if: always()
+        run: |
+          mkdir -p bundles /tmp/reports
+          find bundles -path '*/root/*overlay2' -prune -o -type f \( -name '*-report.json' -o -name '*.log' -o -name '*.out' -o -name '*.prof' -o -name '*-report.xml' \) -print | xargs sudo tar -czf /tmp/reports.tar.gz
+          tar -xzf /tmp/reports.tar.gz -C /tmp/reports
+          sudo chown -R $(id -u):$(id -g) /tmp/reports
+          tree -nh /tmp/reports
```

Comment:
perhaps we need a make target or script for this at some point 😂  - it looks to be repeated multiple times

also not for this PR: wondering if we can fix the permissions when the file is generated (so that we don't have to do the `chown`)

Reply:
Feel strange to have a make target for ci stuff. Maybe a reusable step would do the trick. Do you want that in a follow-up or here?

> also not for this PR: wondering if we can fix the permissions when the file is generated (so that we don't have to do the `chown`)

we do the same in Jenkins: https://github.com/moby/moby/blob/master/Jenkinsfile#L136 so smth to fix in follow-up maybe